internal/io/http/client.go

```diff
@@ -45,17 +45,19 @@ type ClientConf struct {
 }
 
 type RawConf struct {
-	Url                string      `json:"url"`
-	Method             string      `json:"method"`
-	Body               string      `json:"body"`
-	BodyType           string      `json:"bodyType"`
-	Headers            interface{} `json:"headers"`
-	InsecureSkipVerify bool        `json:"insecureSkipVerify"`
-	CertificationPath  string      `json:"certificationPath"`
-	PrivateKeyPath     string      `json:"privateKeyPath"`
-	RootCaPath         string      `json:"rootCaPath"`
-	Timeout            int         `json:"timeout"`
-	DebugResp          bool        `json:"debugResp"`
+	Url                  string      `json:"url"`
+	Method               string      `json:"method"`
+	Body                 string      `json:"body"`
+	BodyType             string      `json:"bodyType"`
+	Headers              interface{} `json:"headers"`
+	InsecureSkipVerify   bool        `json:"insecureSkipVerify"`
+	CertificationPath    string      `json:"certificationPath"`
+	PrivateKeyPath       string      `json:"privateKeyPath"`
+	RootCaPath           string      `json:"rootCaPath"`
+	TLSMinVersion        string      `json:"TLSMinVersion"`
```

Comment:
Can we follow the convention of other properties here. `json:"TLSMinVersion"` ->  `json:"tlsMinVersion"`